index.html

```diff
@@ -2133,10 +2258,40 @@ <h2>
         </dt>
         <dd>
           A boolean that indicates whether the <a>user agent</a> SHOULD collect
-          and return a shipping address as part of the payment request. For
-          example, this would be set to true when physical goods need to be
-          shipped by the merchant to the user. This would be set to false for
-          the purchase of digital goods.
+          and return a shipping address as part of the payment request (e.g.,
+          when physical goods need to be shipped by the payee to the user). The
+          <a>user agent</a> returns the shipping address in the
+          {{PaymentRequest}}'s {{PaymentRequest/shippingAddress}}
+          attribute and the {{PaymentResponse}}'s
+          {{PaymentResponse/shippingAddress}}
+          attribute. The former is subject to a
+          <a data-lt="shipping-redact-list">redact list</a> per the
```

Comment:
See comment below about this list... 

```suggestion
          <a>shipping redact list</a> per the
```